### PR TITLE
SceneObjHolder done

### PR DESCRIPTION
**Pull request by SunakazeKun**
Did all of SceneObjHolder, however, I'm not sure how to translate some of the Japanese names so I left comments for the few.
Also fixed the convertMap script to discard some extra lines. Also, lot's and lots of PAL symbols.

### CODE REVIEW

**Comment by SuperHackio** [comment]
> I'm not sure how to translate some of the Japanese names so I left comments for the few

I got u fam.
(Stuff in brackets indicate what the suffix could be. Example: StageData(Holder) could be StageDataHolder)

47 => SwitchingScreenEffect(Manager)
48 => TimeStopScreenTexture(Holder)
49 => TimeStopScreenEffect(Manager)
50 => SpinSwitch(Controller)                             [Created by SpinSwitchArea and 2 other objects]
51 => WorldMap(Holder)                                  [Created in all worldmap scenes iirc]
52 => WorldMapResult(Controller)                   [Created in Starship Mario iirc]
53 => ScoreAttack(Accessor)                            [Used by the Chimp]
54 => ScoreDisplay(Holder)                             [Also used by the Chimp]
55 => Postman(Controller)
56 => SupportPlay(Controller)                        [I assume this is a controller for the Co-Star luma]

76 => ShadowController(Holder)
77 => ShadowColor(Controller)

92 => SnowStep(Manager)

124 => PlayerFollowPath(Manager?)               [This is the best I can get...]

133 => SwitchingObject(Holder)

137 => PictureBookTeleportDemo(Controller)

148 => Unknown NPC controller
149 => MeisterNotice(Controller)
150 => MeisterEvent(Controller)

Btw you accidentally swapped obj 148 and obj 149 🙃

File: include/Util/ActorAnimUtil.h [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	bool setBpkFrameAndStop(const LiveActor *, f32);
+	bool setBvaFrameAndStop(const LiveActor *, f32);
+	bool setBckFrameEndAndStop(const LiveActor *);
+	bool setBrkFrameEndAndStop(const LiveActor *);
+
+	void startBtkAndSetFrameAndStop(const LiveActor *, const char*, f32);
+	void startBrkAndSetFrameAndStop(const LiveActor *, const char*, f32);
+	void startBtpAndSetFrameAndStop(const LiveActor *, const char*, f32);
+	void startBvaAndSetFrameAndStop(const LiveActor *, const char*, f32);
+	void startBtk(const LiveActor *, const char*);
```
Comment (shibbo) [inline include/Util/ActorAnimUtil.h:49]: spacing on all four const char * pointers

File: include/Util/ActorAnimUtil.h [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	void startBrk(const LiveActor *, const char*);
+	void startBtp(const LiveActor *, const char*);
+	void startBpk(const LiveActor *, const char*);
+	void startBva(const LiveActor *, const char*);
+	bool startBckIfExist(const LiveActor *, const char*);
```
Comment (shibbo) [inline include/Util/ActorAnimUtil.h:54]: same as above

File: include/Util/BaseMatrixFollowTargetHolder.h [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+#include "JGeometry/TPosition3.h"
+#include "JGeometry/TMatrix34.h"
+#include "JMap/JMapInfoIter.h"
+
+class BaseMatrixFollowValidater;
+class LiveActor;
+
+namespace MR {
+	void addBaseMatrixFollowTarget(LiveActor*, const JMapInfoIter&, const JGeometry::TPosition3<JGeometry::TMatrix34<JGeometry::SMatrix34C<f32> > >*, BaseMatrixFollowValidater*);
```
Comment (shibbo) [inline include/Util/BaseMatrixFollowTargetHolder.h:11]: spacing of reference symbol, and you do not need the double spacing between closing template brackets in function declarations

File: include/Util/Functor.h [hunk 4]
```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#import "JKernel/JKRHeap.h"
```
Comment (shibbo) [inline include/Util/Functor.h:3]: #include, not #import

File: include/c_stdlib.h [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+	/* math.h */
+	double acos(double);
+	double asin(double);
+	double atan(double);
+	double atan2(double, double);
+	double ceil(double);
+	double copysign(double, double);
+	double cos(double);
+	double cosh(double); // not in nsmbw
```
Comment (shibbo) [inline include/c_stdlib.h:16]: lol nsmbw header copypaste

File: include/Util/SceneUtil.h [hunk 6]
```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "Actor/LiveActor/LiveActor.h"
+
+namespace MR
+{
+	const char * getCurrentStageName();
```
Comment (shibbo) [inline include/Util/SceneUtil.h:7]: no space between const char*

File: include/Util/ObjUtil.h [hunk 7]
```diff
@@ -1,33 +1,128 @@
...
 
+    void requestAppearPowerStar(const NameObj *, const JGeometry::TVec3<f32> &);
+    void requestAppearPowerStar(const NameObj *, s32, const JGeometry::TVec3<f32> &);
+    void requestAppearPowerStar(const NameObj *, const LiveActor *, f32);
+
+    void requestAppearPowerStarCoin100();
+    bool isEndPowerStarAppearDemo(const NameObj *);
+    NameObj* createPowerStarDemoModel(const NameObj *, const char *, Mtx);
+
+    void declareCoin(const NameObj*, s32);
```
Comment (shibbo) [inline include/Util/ObjUtil.h:86]: NameObj *

File: include/Util/ActorSensorUtil.h [hunk 8]
```diff
@@ -8,17 +8,67 @@ class HitSensor;
 
 namespace MR
 {
-	HitSensor* getSensor(LiveActor *, const char *);
-	void addBodyMessageSensorMapObj(LiveActor *);
-	HitSensor* addHitSensorMapObj(LiveActor *, const char *, u16, f32, const JGeometry::TVec3<f32> &);
-	HitSensor* addHitSensorMapObjMoveCollision(LiveActor *, const char *, u16, f32, const JGeometry::TVec3<f32> &);
+	HitSensor* addHitSensor(LiveActor*, const char*, u32, u16, f32, const JGeometry::TVec3<f32>&);
```
Comment (shibbo) [inline include/Util/ActorSensorUtil.h:11]: const char *

**Pull request merged**